istioctl/cmd/root.go

```diff
@@ -299,3 +299,15 @@ func seeExperimentalCmd(name string) *cobra.Command {
 		},
 	}
 }
+
+// setNotSupportedCmd is used to set commands that are not supported for OpenShift Service Mesh.
+func setNotSupportedCmd(name string) *cobra.Command {
```

Comment:
Why is this called `setNotSupportedCmd`? I think it could simply be called `unsupportedCmd`.

Reply:
Done